cmd/root/root.go

```diff
@@ -155,5 +164,24 @@ Stack Trace:
 		}
 	}
 
+	exitCode := 0
+	if err != nil {
+		exitCode = 1
+	}
+
+	ctx = cmd.Context()
+	telemetryErr := telemetry.Upload(ctx, protos.ExecutionContext{
+		CmdExecID:       command.ExecId(ctx),
+		Version:         build.GetInfo().Version,
+		Command:         commandString(cmd),
+		OperatingSystem: runtime.GOOS,
+		DbrVersion:      dbr.RuntimeVersion(ctx),
+		ExecutionTimeMs: time.Since(startTime).Milliseconds(),
+		ExitCode:        int64(exitCode),
+	})
+	if telemetryErr != nil {
+		log.Debugf(ctx, "telemetry upload failed: %s", telemetryErr)
```

Comment:
Should we raise the level of this? e.g. to Infof. It does not matter now but the plan is to have -v/-q so it's good have appropriate levels.